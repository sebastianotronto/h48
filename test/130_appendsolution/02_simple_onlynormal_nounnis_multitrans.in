U F R

0
4
rotation UF
mirrored UR
rotation BD
mirrored LB
