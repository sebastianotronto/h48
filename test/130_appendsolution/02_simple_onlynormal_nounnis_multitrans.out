U F R
B D L
U' L' B'
R' B' D'
Number of solutions: 4
Shortest solution length: 3
Used bytes: 30
